src/TSTransformer/nodes/expressions/transformIdentifier.ts

```diff
@@ -11,14 +11,17 @@ import { getFirstConstructSymbol } from "TSTransformer/util/types";
 import ts from "typescript";
 
 export function transformIdentifierDefined(state: TransformState, node: ts.Identifier) {
-	const symbol = ts.isShorthandPropertyAssignment(node.parent)
-		? state.typeChecker.getShorthandAssignmentValueSymbol(node.parent)
-		: state.typeChecker.getSymbolAtLocation(node);
-	assert(symbol);
-
-	const replacementId = state.symbolToIdMap.get(symbol);
-	if (replacementId) {
-		return replacementId;
+	// synthetic nodes don't have parents
```

Comment:
What would be the synthetic node(s) encountered here?
(It may be useful to add the answer to this question as a comment)